src/util/fd_util.c

```diff
@@ -10,9 +6,6 @@ fd_boot( int *    pargc,
   /* At this point, we are immediately after the program start, there is
      only one thread of execution and fd has not yet been booted. */
   fd_log_private_boot  ( pargc, pargv );
-# if defined(__linux__)
-  fd_linux_private_boot( pargc, pargv );
-# endif
```

Comment:
Are you sure we want to get rid of this? Do we ever run unit tests in a network namespace?